edward/inferences/replica_exchange_mc.py

```diff
@@ -54,26 +54,23 @@ def __init__(self, latent_vars, proposal_vars, data=None,
     """
     check_latent_vars(proposal_vars)
     self.proposal_vars = proposal_vars
+    super(ReplicaExchangeMC, self).__init__(latent_vars, data)
 
     self.n_replica = len(inverse_temperatures)
     if inverse_temperatures[0] != 1:
       raise ValueError("inverse_temperatures[0] must be 1.")
-    self.inverse_temperatures = [tf.convert_to_tensor(inverse_temperature,
-                                 dtype=list(latent_vars.values())[0].dtype)
-                                 for inverse_temperature in
-                                 inverse_temperatures]
+    self.inverse_temperatures = tf.constant(
```

Comment:
This fails if `inverse_temperatures` is already a Tensor. Maybe cast?

Reply:
Thank you for the advice. I'll fix.